Resources/Audio/Effects/Gasp/attributions.yml

```diff
@@ -24,3 +24,9 @@
   license: "CC-BY-SA-3.0"
   copyright: "Taken from tgstation at https://github.com/tgstation/tgstation/commit/f7a49c4068f1277e6857baf0892d355f1c055974"
   source: "https://github.com/tgstation/tgstation/tree/f7a49c4068f1277e6857baf0892d355f1c055974/sound/voice/human"
+
+- files:
+  - moth_DeathGasp.ogg
+  license: "CC 0 license"
```

Comment:
How can I fix this?

Reply:
You can see the valid license identifiers here: https://github.com/space-wizards/RobustToolbox/blob/master/Schemas%2Frga_validators.py